webpack/react_app/components/HostKebab/KebabItems.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { DropdownItem } from '@patternfly/react-core';
+import { CodeIcon } from '@patternfly/react-icons';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { HOST_DETAILS_KEY } from 'foremanReact/components/HostDetails/consts';
+
+const HostKebabItems = () => {
+  const { cockpit_url: consoleUrl } = useSelector(state =>
```

Comment:
There is no trivial way to propagate the `cockpit_url` from core,  for an easy cherry-pick the attribute is taken directly from redux's store.

Reply:
In this case it's IMHO fine as it is not one of main attributes :)